### ISSUE TITLE
date_from/date_to not handled correctly in get_creditcard_transactions

### ISSUE THREAD

**Issue by githubuser3000** [grindsa/dkb-robo#22]
In function get_creditcard_transactions:
radio button filterType is set by the code correctly to:
```
self.dkb_br["filterType"] = 'DATE_RANGE'
self.dkb_br["postingDate"] = str(date_from)
self.dkb_br["toPostingDate"] = str(date_to)
```

but return values are always, like it has been set to:
```
self.dkb_br["filterType"] = 'PERIOD'
self.dkb_br["slSearchPeriod"] = 0
```

As result, the transactions returned by DKB are always since last balance accounting. date_from is not considered.

A code review shows no errors, however the results are wrong. Can anybody confirm this behavior?

**Comment by githubuser3000** [comment]
I figured out the root cause. The problem occurs, if the date_to is in the future.

- The form for account transaction has a benign reaction to this case and automatically corrects the date to _today_.
- The form for creditcard transactions however reacts with an error and falls back to "filterType" = 'PERIOD'

A solution would be to always max the date_to to _today_.

**Comment by grindsa** [comment]
"A solution would be to always max the date_to to today." I think that makes sense and I believe it can be done in the library itself. Let me look into it over the weekend....

**Comment by grindsa** [comment]
included in v0.11. Thus, I am closing this issue...
